llvm/include/llvm/IR/IntrinsicsNVVM.td

```diff
@@ -5043,6 +5043,16 @@ def int_nvvm_prefetch_global_L2_evict_last: Intrinsic<[], [llvm_global_ptr_ty],
 def int_nvvm_prefetchu_L1 : Intrinsic<[], [llvm_ptr_ty],
   [IntrArgMemOnly, ReadOnly<ArgIndex<0>>, NoCapture<ArgIndex<0>>]>;
 
+def int_nvvm_applypriority_global_L2_evict_normal
+  : Intrinsic<[], [llvm_global_ptr_ty, llvm_i64_ty],
```

Comment:
Can these be made DefaultAttrsIntrinsics? If not can some of the default attrs such as IntrWillReturn be applied?

Reply:
Yes addressed.